Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -419,18 +377,10 @@ - (void)videoPlaybackResumed:(NSDictionary *)properties withOptions:(NSDictionar
 
 - (void)videoContentStarted:(NSDictionary *)properties withOptions:(NSDictionary *)integrations
 {
-    NSDictionary *map = returnMappedContentProperties(properties, integrations);
-    [[self.streamAnalytics playbackSession] setAssetWithLabels:map];
-    SEGLog(@"[[SCORStreamingAnalytics playbackSession] setAssetWithLabels:%@", map);
-
-    if ([properties[@"position"] longValue]) {
-        long playPosition = [properties[@"position"] longValue];
-        [self.streamAnalytics notifyPlayWithPosition:playPosition];
-        SEGLog(@"[[SCORStreamingAnalytics streamAnalytics] notifyPlayWithPosition:%ld", playPosition);
-    } else {
-        [self.streamAnalytics notifyPlay];
-        SEGLog(@"[[SCORStreamingAnalytics streamAnalytics] notifyPlay;");
-    }
+    [self addMetaDataAssets:properties withOptions:integrations];
+    [self movePosition:properties];
```

Comment:
Since this is considered a start event do we need the position here?

Reply:
Since it originally called  ` [self.streamAnalytics notifyPlayWithPosition:playPosition];`, we need to call both movePosition and and startEvent.